### PR TITLE
Live tests/v2

### PR DESCRIPTION
**Pull request by victorjulien**
Expand live tests with various unix socket commands. Fix bugs found while doing this:
https://redmine.openinfosecfoundation.org/issues/7020
https://redmine.openinfosecfoundation.org/issues/7022
Clean version of #11061

### CODE REVIEW

**Review by catenacyber — APPROVED** [review]
Thanks for the work :-)

- CI : 🟢 
- Code : good
- Commits segmentation : nice
- Commit messages : ok for me, do we put dots after ticket numbers ?
- Git ID set : looks fine for me
- CLA : you already contributed :-p
- Doc update : not needed
- Redmine tickets : ok
- Rustfmt : no rust
- Tests : nice, maybe some more comments
- Dependencies added: none

File: .github/workflows/live/afp-ids.sh [hunk 1]
```diff
@@ -67,18 +68,73 @@ if [ $SID1CHECK = false ]; then
...
 fi
+CHECK=$(jq -c 'select(.alert.signature_id == 222)' ./eve.json | wc -l)
+if [ $CHECK -ne 1 ]; then
+	echo "ERROR alerts count off for sid 222 (datasets)"
+    RES=1
+fi
+JSON=$(python3 python/bin/suricatasc -v -c "dataset-clear ipv4-list ipv4" /var/run/suricata/suricata-command.socket)
+echo $JSON
+
+sleep 5
```
Comment (catenacyber) [inline .github/workflows/live/afp-ids.sh:79]: What is happening in these 5 seconds ?

Comment (victorjulien) [reply .github/workflows/live/afp-ids.sh:79]: just mean to give the dataset clear to have the effect of allowing another alert for 222

Comment (catenacyber) [reply .github/workflows/live/afp-ids.sh:79]: Gives food to think about the best framework for testing Suricata's features that only happen live like suricatasc...

**Comment by victorjulien** [comment]
Merged in #11076, thanks!

**Pull request closed without merging**